README.md

```diff
@@ -89,12 +89,24 @@ autoinstall:
       printf "network:\n  version: 2\n  renderer: NetworkManager"
       > /target/etc/netplan/01-network-manager-all.yaml
 
+    # disable systemd-networkd and associated units
+    # On desktop, NetworkManager is the default network stack
+    # https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/2036358
+    - curtin in-target -- systemctl disable systemd-networkd.service
+
     # Remove default filesystem and related tools not used with the suggested
     # 'direct' storage layout.  These may yet be required if different
     # partitioning schemes are used.
     - >-
       curtin in-target -- apt-get remove -y
       btrfs-progs cryptsetup* lvm2 xfsprogs
+    # https://github.com/canonical/autoinstall-desktop/issues/9
+    # - curtin in-target -- apt-get install -y cryptsetup lvm2 btrfs-progs xfsprogs
```

Comment:
shouldn't this be present after ubuntu-server is removed?

Reply:
Yes, correct, in case of `- curtin in-target -- apt-get install -y cryptsetup ...` your should place this after `- curtin in-target -- apt-get remove -y ubuntu-server ubuntu-server-minimal...`